app/services/course_search_service.rb

```diff
@@ -7,12 +7,12 @@ def initialize(
     filter:,
     sort: nil,
     course_scope: Course,
-    course_type_answer_determiner: Find::CourseTypeAnswerDeterminer
+    degree_type_answer_determiner: Find::CourseTypeAnswerDeterminer
```

Comment:
Should we renamed `CourseTypeAnswerDeterminer` to degree type?

Reply:
Good shout. Updated here: https://github.com/DFE-Digital/publish-teacher-training/pull/4501/commits/e56faeee8f4a38bdf40fffa41d6a4278c22f155d